src/api/store/userprofile.py

```diff
@@ -205,20 +205,21 @@ def create_user(self, context: Context, args) -> (dict, MassEnergizeAPIError):
 
       if not email:
         return None, CustomMassenergizeError("email required for sign up")
-      
       user = UserProfile.objects.filter(email=email).first()
+      print("this is the user")
+      print(user)
       if not user:
         new_user: UserProfile = UserProfile.objects.create(
           full_name = args.get('full_name'), 
           preferred_name = args.get('preferred_name', None), 
           email = args.get('email'), 
           is_vendor = args.get('is_vendor', False), 
-          accepts_terms_and_conditions = args.pop('accepts_terms_and_conditions', False)
+          accepts_terms_and_conditions = args.pop('accepts_terms_and_conditions', False), 
+          color = args.get('color')
```

Comment:
This will blow up if 'color' not specified, which can be the case.
For args which are required, check out the self.validator.expect('argname', argtype, required=True) in the handler routine.